Corgibytes.Freshli.Cli/Commands/DoctorCommand.cs

```diff
@@ -0,0 +1,22 @@
+using System.CommandLine;
+using Corgibytes.Freshli.Cli.CommandOptions;
+
+namespace Corgibytes.Freshli.Cli.Commands;
+
+public class DoctorCommand : RunnableCommand<DoctorCommand, DoctorCommandOptions>
+{
+    public DoctorCommand() : base("doctor",
+        "Validating environment assumptions.")
+    {
+        var gitPath = new Option<string>(
+            "--git-path",
+            description: "Path to the git binary. Default = 'git'",
+            getDefaultValue: () => "git"
```

Comment:
```suggestion
            getDefaultValue: () => configuration.GitPath
```

We ideally re-use the configuration object here to give us the default path to git. [AnalyzeCommand](https://github.com/corgibytes/freshli-cli/blob/main/Corgibytes.Freshli.Cli/Commands/AnalyzeCommand.cs#L9) does that for example.